frontend/src/app/modules/filter-section/filter-section-routing.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
+import { RouterModule, Routes } from '@angular/router';
+import { FilterSectionComponent } from '@modules/filter-section/filter-section/filter-section.component';
+
+const routes: Routes = [
+    {
+        path: '',
+        component: FilterSectionComponent,
+    },
+];
+
+@NgModule({
+    imports: [RouterModule.forChild(routes)],
+    exports: [RouterModule],
+})
```

Comment:
Why you created a separate module for this?

Reply:
And should it be here?

Reply:
> And should it be here?

Should it be in different place? I placed this component and modules to another components and modules

Reply:
I mean, that you don't move it from one place to another
From code perspective, you just created new one (Or it was created?)

Reply:
> I mean, that you don't move it from one place to another From code perspective, you just created new one (Or it was created?)

Probably, but I don't know how, because all the files that was in the project I just moved, but somehow git shows me half of them as modified and other half as new ones, I thought that it's strange, but I didn't think that it will create some issues. And about that routing module, you're right, I this component shouldn't have this module